charts/devtron/templates/NOTES.txt

```diff
@@ -7,4 +7,23 @@
 2. You can watch the progress of Devtron microservices installation by the following command
 
    kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync.status}'
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+3. You can access devtron dashboard with following.
```

Comment:
Instead of starting point number 3 here, just add number 3 before the address line